packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/gateway/TokenGateway.sol

```diff
@@ -19,9 +19,10 @@
 pragma solidity ^0.6.11;
 
 import "./ITokenGateway.sol";
+import "../ERC165.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
 
-abstract contract TokenGateway is ITokenGateway {
+abstract contract TokenGateway is ITokenGateway, ERC165 {
```

Comment:
hm we won't upgrade the L2 contracts, so maybe we don't want this interface to inherit ERC165 for now